.eslint-rules/globals.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  // Globals can be disabled with the string "off"
+  // "writable" to allow the variable to be overwritten or "readonly" to disallow overwriting.
+  globals: {
+    Atomics: "readonly",
+    SharedArrayBuffer: "readonly",
+    // Makes logger function available everywhere. Else eslint will complaint of undef-var.
+    logger: "readonly",
+    module: "writable",
+    // Makes props obtained from Rails backend available everywhere in this project.
```

Comment:
Should I remove this? Since we won't have anything related to Rails backend.

Reply:
Yes